nise/yaml_generators/aws/generator.py

```diff
@@ -48,72 +50,27 @@
 }
 
 
-def generate_words(config):
-    """
-    Generate a hyphen-separated string of words.
-
-    The number of words is specified in the config. (config.max_name_words)
-    """
-    return "-".join(FAKER.words(config.max_name_words))
-
-
-def generate_number_str(config):
-    """
-    Generate a string of digits of arbitrary length.
-
-    The maximum length is specified in the config. (config.max_resource_id_length)
-    """
-    return str(FAKER.random_int(0, 10 ** config.max_resource_id_length)).zfill(config.max_resource_id_length)
-
-
-def generate_name(config, prefix="", suffix="", dynamic=True, generator=generate_words, cache=SEEN_NAMES):
-    """
-    Generate a random resource name using faker.
-
-    Params:
-        config : dicta - config information for the generator
-        prefix : str - a static prefix
-        suffix : str - a static suffix
-        dynamic : bool - flag to run the generator function
-        generator : func - function that will generate the dynamic portion of the name
-        cache : set - a cache for uniqueness across all calls
-
-    Returns:
-        str
-    """
-    new_name = None
+def uniform_yield(a, b):
+    """Yield from random.uniform."""
     while True:
-        if prefix:
-            prefix += "-"
-        if suffix:
-            suffix = "-" + suffix
-        mid = generator(config) if dynamic else ""
-        new_name = f"{prefix}{mid}{suffix}"
-        if new_name not in cache:
-            cache.add(new_name)
-            break
+        yield uniform(a, b)
 
-    return DBL_DASH.sub("-", new_name)
 
+RATE_AMT = {
+    "DTG": (uniform_yield(0.12, 0.19), uniform_yield(0.000002, 0.09)),
+    "EBS": (uniform_yield(0.02, 0.16), uniform_yield(0.2, 300.99)),
+    "S3": (uniform_yield(0.02, 0.06), uniform_yield(0.2, 6000.99)),
+}
 
-def generate_resource_id(config, prefix="", suffix="", dynamic=True):
-    """
-    Generate a random resource id using faker.
 
-    Params:
-        config : dicta - config information for the generator
-        prefix : str - a static prefix
-        suffix : str - a static suffix
-        dynamic : bool - flag to run the generator function
-        generator : func - function that will generate the dynamic portion of the resource id
-        cache : set - a cache for uniqueness across all calls
+def product_sku_generator(*args):
+    """Generate random 12 character alpha string."""
+    return FAKER.pystr(min_chars=12, max_chars=12).upper()
 
-    Returns:
-        str
-    """
-    return generate_name(
-        config, prefix=prefix, suffix=suffix, dynamic=dynamic, generator=generate_number_str, cache=SEEN_RESOURCE_IDS
-    )
+
+def resource_id_generator(*args):
+    """Generate fake.ean8() number."""
+    return FAKER.ean8()
```

Comment:
This function is pointless. Instead of calling this function, the code should just call FAKER.ean8() directly.